### ISSUE TITLE
Have display not duplicate plots in windows

### ISSUE THREAD

**Issue by spencerlyon2** [spencerlyon2/PlotlyJS.jl#20]
Right now the main display function looks like this:

```julia
function Base.display(p::Plot)
    w = get_window(p)
    Blink.load!(w, _js_path)
    @js w begin
        @var thediv = document.createElement("div")
        thediv.id = $(string(p.divid));
        document.body.appendChild(thediv);
        Plotly.newPlot(thediv, $(p.data),  $(p.layout), d("showLink"=> false))
    end
    show(p)
end
```

This will create a new div with the same id each time the plot is re-displayed -- causing copies of the plot to show up in the window. 

We need some logic here that says create the div if it doesn't exist, or just grab the existing one.

Anyone have any ideas?

cc: @one-more-minute

**Comment by one-more-minute** [comment]
Perhaps call `body!(w, "")` to reset things before adding the plot?

**Comment by spencerlyon2** [comment]
That would definitely work, but I'd like to be able to leverage plotly's (somewhat) efficient updating mechanisms. 

I know we lose a bit of performance relative to a js app because we have the rpc layer in between, but I would think that wiping body and then starting fresh will make it much slower than it needs to be.

It seems like this should be really easy:

```javascript
thediv = document.getElementbyID("$(p.divid)")
if thediv == null
  thediv = document.createElement("div")
  thediv.id = $(string(p.divid));
  document.body.appendChild(thediv);
end
```

but that doesn't work because blink isn't set up to handle `if` yet.

**Comment by one-more-minute** [comment]
Yeah that's fair, I didn't realise you were hoping to modify the original plot. Try [this](https://github.com/JunoLab/Blink.jl/commit/1df3a72ba40e48a4ee44755b1b828cde552bbf64) out, I haven't tested it really thoroughly but it should do the trick.

**Comment by spencerlyon2** [comment]
Closed by 22a901e818811a4f51f8cd73963e083742987807

I needed to teach blink a few more tricks to be able to handle this...
